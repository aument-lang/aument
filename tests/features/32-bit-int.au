print 1234567;