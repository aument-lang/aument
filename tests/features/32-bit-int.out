int;1234567